_includes/selector.html

```diff
@@ -456,7 +456,10 @@
                 };
             },
             getAdditionalPkgName(pkg) {
+                // used to pin the version of nx-cugraph for conda install
+                var rapids_version = this.active_release === "Stable" ? this.getStableVersion() : this.getNightlyVersion();
                 var pkg_names = {
+                    "NetworkX + nx-cugraph": `networkx nx-cugraph=${rapids_version}`,
```

Comment:
This is annoying... the nx-cugraph version pinning is highlighted but the normal version pins are not. 
![image](https://github.com/user-attachments/assets/5d46959a-2284-45ec-b3c0-aba6e624e3e5)

I'm willing to accept this as-is but we could refactor the block below so it only highlights the package name and not the pinning:
```javascript
if (this.active_additional_packages.length) {
    add_pkgs_ln = indentation + this.active_additional_packages
    .map(pkg => this.highlightCondaPkg(this.getAdditionalPkgName(pkg)))
        .join(" ");
}
```

Reply:
@bdice Would it be alright to file an issue for this and refactor the function that highlights package names in a PR down the road?

Reply:
Yes -- that's totally fine.